core/src/main/java/org/apache/metamodel/query/SelectItem.java

```diff
@@ -190,6 +190,17 @@ public SelectItem(FunctionType function, Object[] functionParameters, Column col
     }
 
     /**
+     * Creates a SelectItem that uses a function on a column from a particular
```

Comment:
Since this constructor does not pass in any ```FromItem``` then I guess it is not a "particular FromItem" but the opposite.